tony-core/src/main/java/com/linkedin/tony/Constants.java

```diff
@@ -67,6 +67,7 @@
   public static final String HADOOP_CONF_DIR = ApplicationConstants.Environment.HADOOP_CONF_DIR.key();
   public static final String TONY_SITE_CONF = "tony-site.xml";
   public static final String TONY_CONF_DIR = "TONY_CONF_DIR";
+  public static final String DEFAULT_TONY_CONF_DIR = "/export/apps/tony/conf";
```

Comment:
it is weird to put this as the default value

Reply:
Currently, the `TONY_CONF_DIR` environment variable is not set by default. It might be a while until Grid SRE sets this on all client machines by default. Until then, I want users to automatically pick up `/export/apps/tony/conf/tony-site.xml` (which I've installed manually for now), so that history files get written out to the correct location.

Note that even if `/export/apps/tony/conf` does not exist on the client machine, there will not be any runtime exceptions.